enclave_manager/avalon_enclave_manager/wpe/wpe_config.toml

```diff
@@ -0,0 +1,112 @@
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[KvStorage]
+# the remote version is of higher priority if enabled
+remote_url = "http://localhost:9090"
+
+# --------------------------------------------------
+# Logging -- configuration of service logging
+# --------------------------------------------------
+[Logging]
+LogLevel = "INFO"
+LogFile  = "__screen__"
+
+# ------------------------------------------------------------------
+# Work load execution-settings for workload execution(synchronous/asynchronous)
+# ------------------------------------------------------------------
+[WorkloadExecution]
+# Set this flag to 1 for sync workorder execution.Else set it to 0
+sync_workload_execution = 1
+
+# -------------------------------------------------------------------
+# EnclaveManager -- configuration of Intel SGX Enclave EnclaveManager
+# -------------------------------------------------------------------
+[EnclaveManager]
+# configurations for asynchronous work load execution
+sleep_interval = "10"
+# configurations for synchronous Work load execution
+# zmq_port is the port used for zmq socket communication
+# Port for socket interaction with listener
+zmq_port = '5555'
+
+[Listener]
+listener_port = 1947
+max_work_order_count = 10
+
+# -------------------------------------------------------
+# EnclaveModule -- configuration of the Intel SGX Enclave
+# -------------------------------------------------------
+[EnclaveModule]
+# Service Provider ID (SPID) is a 32-digit hex string tied to the
+# enclave implementation. Replace dummy SPID with value obtained after
+# subscription to run TCS in Intel SGX HW mode.
+spid = "DEADBEEF00000000DEADBEEF00000000"
+
+num_of_enclaves = "1"
+
+# ias_url is the URL of the Intel Attestation Service (IAS) server.
+ias_url = "https://api.trustedservices.intel.com/sgx/dev"
+
+# Proxy for https. Leave commented out for direct Internet connections or
+# uncomment and change to your corporate proxy.
+#https_proxy = "http://your-proxy:your-port/"
+
+# IAS API key is a 32-digit hex string subscription key used for authentication
+# of requests submitted to the IAS server. Obtain the key by subscribing in
+# the portal https://api.portal.trustedservices.intel.com/
+ias_api_key = "112233445566778899aabbccddeeff00"
+
+# TEE enclave library to use.
+enclave_library = "libtcf-enclave.signed.so"
+enclave_library_path = "tc/sgx/trusted_worker_manager/enclave/build/lib/"
+
+# -------------------------------------------------------------
+# TCS Worker configuration details
+# -------------------------------------------------------------
+
+[KMEListener]
+# URL of KMEListener where a Work order processor can send requests meant for
+# the Key Management Enclave(KME)  
+bind = "http://kme-listener:1948" 
+
```

Comment:
Rest of parameters below are not required for now in WPE. Only KME will create worker details. You could move these params to KME config file

Reply:
"KME will create worker details" means it will create a worker keys and also update worker details field in lmdb right? In that case KME is acting as worker manager - creating worker, updating key, etc. Then why should we call it as KME?